### ISSUE TITLE
Join Us Page: Update alt text for the donation chart image on the "Help Us Keep the Lights On" card to adhere to WCAG

### ISSUE THREAD

**Issue by d-perez-8** [hackforla/website#3856]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Join Us` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_includes/donate-card.html`:

**From:**
```
alt="Hosting, mailing list services, Zoom accounts, domain names, stickers, t-shirts"
```

**To:**
```
alt="Monthly: Hosting, Mailing List services, zoom; Annual: Domain Names; Occasional: Stickers, T-shirts"
```

- [x] Ensure that the corresponding `Join Us` webpage stays the same after the change
- [x] Using developer tools to inspect the image, ensure that the new alt text gets incorporated after the change.
  - **Note:** `alt=""` in code while show up as `alt` when using developer tools to inspect the image's alt text property.

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3858 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- Join Us Webpage: https://www.hackforla.org/join

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_includes/donate-card.html`
- See the source file code on the repo: https://github.com/hackforla/website/blob/gh-pages/_includes/donate-card.html
- This issue is part of epic #3858

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by mjshelton12** [comment]
Availability: 6hrs a week with time Sunday and weekday evenings
ETA: 1 week

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-merge ~Please make a new issue to update it to ~

```
alt="Monthly: Hosting, Mailing List services, Zoom, Meetup; Annual: Domain Names, Insurance; Occasional: Stickers, T-shirts"
```

We have to update the image to have more items in it.  Since we don't have a designer right now, what do you think about making the circles with css instead of using images?  The text for each would be the following


Monthly: Hosting, Mailing List services, Zoom, Meetup
Annual: Domain Names, Insurance
Occasional: Stickers, T-shirts

**Comment by ExperimentsInHonesty** [comment]
- A new ER has been created. https://github.com/hackforla/website/issues/4241
